spec/features/users/sign_up_spec.rb

```diff
@@ -11,4 +11,43 @@
       expect(page).not_to have_content t('errors.messages.confirmation_invalid_token')
     end
   end
+
+  context 'with js', js: true do
+    context 'sp loa1' do
+      it 'allows the user to toggle the modal' do
+        sign_up_with_sp
+        expect(page).to have_xpath("//div[@id='sp-loa-cancel'][@class='hide']")
+
+        click_on t('links.cancel')
+        expect(page).to have_xpath("//div[@id='sp-loa-cancel']")
+
+        click_on t('loa1.buttons.continue')
+        expect(page).to have_xpath("//div[@id='sp-loa-cancel'][@class='hide']")
+      end
+
+      it 'allows the user to delete their account and returns them to the home page' do
+        user = sign_up_with_sp
+        click_on t('links.cancel')
+
+        click_on t('loa1.buttons.cancel')
+
+        expect(page).to have_content t('loa1.cancel.success')
+        expect(page).to have_content(t('headings.sign_in_with_sp',
+                                       sp: 'Your friendly Government Agency'))
+        sign_in_user(user)
+
+        expect(page).to have_content t('devise.failure.invalid')
```

Comment:
I believe what we're testing here is that the user is indeed deleted. If so, I would replace these 2 last lines with `expect { User.find(user.id) }.to raise_error ActiveRecord::RecordNotFound` to save some test execution time.